.github/workflows/build.yml

```diff
@@ -21,22 +21,19 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java: [ 11, 13, 15, 16 ]
```

Comment:
Do not remove 11 as yet. Have that also

Reply:
I reference to the previous design as:

```toml
      matrix:
        java: [ 11, 13, 15, 16 ]
        spring-boot-version: [ 2.7.8 ]
        experimental: [ false ]
        include:
          - java: 11
            spring-boot-version: 2.6.14
            experimental: false
          - java: 11
            spring-boot-version: 2.5.7
            experimental: false
          - java: 11
            spring-boot-version: 2.4.0
            experimental: false
          - java: 11
            spring-boot-version: 2.3.6.RELEASE
            experimental: false
```

Through java8 is supported, sdk-tests subproject used jdk11 and set compile source and target to jdk11, and test with springboot 2.x versions with JDK11 as list above.

Reply:
If a jdk version below jdk17 appears here, it means that no jdk17 features can be used in the sdk-tests sub-project, or it will fail to compile.

This means that the sdk-tests project has to be split into two sub-projects, jdk17 and jdk11, and the various test cases have to be rewritten in both sub-projects. Otherwise it would not be covered by both jdk17 and jdk11.

Reply:
That's why it costs me so much to keep the sdk-tests subproject from being split.

Reply:
@skyao For now we are not using any jdk17 features in the idk-tests sub-project right? 
And as long as we support jdk8 and jdk11, won't that continue?